src/lib/common/string.cpp

```diff
@@ -676,3 +676,41 @@ void strReplace(char* to, int toLen, const char* from, const char* oldString, co
 
   to[toIx] = 0;
 }
+
+
+
+/* ****************************************************************************
+*
+* servicePathCheck - 
+*/
+std::string servicePathCheck(const char* servicePath)
```

Comment:
Not sure you should unify these two functions. They are very different.
I would leave them as is.
Comments are always good, of course.